database/postgres/postgres.go

```diff
@@ -320,7 +320,7 @@ func (p *Postgres) Drop() error {
 	if len(tableNames) > 0 {
 		// delete one by one ...
 		for _, t := range tableNames {
-			query = `DROP TABLE IF EXISTS ` + t + ` CASCADE`
+			query = `DROP TABLE IF EXISTS "` + t + `" CASCADE`
```

Comment:
Use [pq.QuoteIdentifier()](https://godoc.org/github.com/lib/pq#QuoteIdentifier) instead

Reply:
:+1: I didn't realize `lib/pq` had that.

Will update the other places as well.